source/termination_criteria/steady_heat_flux.cc

```diff
@@ -60,7 +60,7 @@ namespace aspect
 
       for (const auto &cell : this->get_dof_handler().active_cell_iterators())
         if (cell->is_locally_owned())
-          for (unsigned int f=0; f<GeometryInfo<dim>::faces_per_cell; ++f)
+          for (const unsigned int f : cell->face_indices())
```

Comment:
I really like that these can be const now. :+1:

Reply:
Yes :-)